packages/lib-classifier/src/store/Project/Project.js

```diff
@@ -0,0 +1,24 @@
+import { types } from 'mobx-state-tree'
+import Resource from '../Resource'
+
+const Project = types
+  .model('Project', {
+    configuration: types.frozen({}),
+    display_name: types.string,
+    experimental_tools: types.frozen([]),
+    links: types.frozen({}),
+    slug: types.string
+  })
+  .views(self => ({
+    get defaultWorkflow() {
+      const activeWorkflows = self.links['active_workflows']
+      let singleActiveWorkflow
+      if (activeWorkflows.length === 1) {
+        [singleActiveWorkflow] = self.links['active_workflows']
+      }
+      const defaultWorkflow = self.configuration['default_workflow']
```

Comment:
I didn't get a chance to leave this comment before this got merged, but we should be checking that the default workflow is active. The lab UI prevents a default being set if it's inactive, but if there's a bug in the lab or if a project owner somehow manages to configure an inactive workflow to be default some other way (maybe python client), then we have an issue. We should validate its activeness.

Reply:
This line's been removed now. When a project has multiple workflows, a volunteer has to specify which one they're going to classify on. The classifier won't fall back to a project default.